src/utils/misc.ts

```diff
@@ -0,0 +1,4 @@
+/**
+ * `true` if device is an iOS device, else `false`
+ */
+export const isIOS = () => /iPad|iPhone|iPod/.test(navigator.userAgent);
```

Comment:
@rithviknishad we don't face the problems with macs right?

Reply:
@skks1212 No issues with MacOS.